packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -1139,6 +1139,23 @@ new ecs.Ec2Service(this, 'EC2Service', {
 });
 ```
 
+### Cluster Default Provider Strategy
+
+When the service does not have a capacity provider strategy, the cluster's default capacity provider strategy will be used.
+Default Capacity Provider Strategy can be added by using the method addDefaultCapacityProviderStrategy.
```

Comment:
Please mention both the fact that you can specify capacity providers in the constructor and with the method, and some of the limitations of the first approach (e.g. you cannot set a default strategy with any ec2 capacity providers by using the constructor).

We should also show an example of creating an ec2 ASG capacity provider and adding it and fargate spot with the method. The current example is invalid, since we do not show `capacityProvider` being added to the cluster's capacity providers.